rest/oidc_api_test.go

```diff
@@ -416,7 +416,7 @@ func TestGetOIDCCallbackURL(t *testing.T) {
 			providers := auth.OIDCProviderMap{"foo": mockProvider("foo"), "bar": mockProvider("bar")}
 			openIDConnectOptions := auth.OIDCOptions{Providers: providers, DefaultProvider: base.StringPtr("foo")}
 			rtConfig := RestTesterConfig{DatabaseConfig: &DatabaseConfig{DbConfig: DbConfig{OIDCConfig: &openIDConnectOptions}}}
-			rt := NewRestTester(t, &rtConfig)
+			rt := NewRestTesterDefaultCollection(t, &rtConfig)
```

Comment:
What pending ticket requires this to be default only?

Reply:
This was unnecessary, removed.